lib/services/third_party_service/multi_media_pick_service.dart

```diff
@@ -0,0 +1,35 @@
+/* This is an abstraction service for picking up Photos/videos
+Library used: image_picker (https://pub.dev/packages/image_picker)
+Service usage: "add_post_view_model.dart"
+*/
+
+import 'dart:async';
+import 'dart:io';
+import 'package:image_picker/image_picker.dart';
+
+class MultiMediaPickerService {
+  MultiMediaPickerService() {
+    _picker = ImagePicker();
+    _fileStream = _fileStreamController.stream.asBroadcastStream();
+  }
+
+  //Local Variables
+  final StreamController<File> _fileStreamController = StreamController();
+  late Stream<File> _fileStream;
+  late ImagePicker _picker;
+
+  //Getters
+  Stream get fileStream => _fileStream;
+
+  Future getPhotoFromGallery({bool camera = false}) async {
```

Comment:
return Future<File>

Reply:
There can be a condition where we're not able to select a file, in that case we need to return "null" to the viewModel, 
If I keep return type as File I'll not be able to return null from this function.

Reply:
Oh this is null safe? return 'Future<File?>' should work